ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,160 @@
+                                    name="./orientation"
+                                    text="Item Alignment">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <horizontal
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="granite/ui/components/coral/foundation/form/radio"
+                                            checked="true"
+                                            text="Horizontal"
+                                            value="horizontal"/>
+                                    <vertical
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="granite/ui/components/coral/foundation/form/radio"
+                                            text="Vertical"
+                                            value="vertical"/>
+                                </items>
+                            </alignment>
+                            <value
+                                    wrapperClass="cmp-adaptiveform-checkboxgroup__value"
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
+                                    fieldDescription="Enter Default Values"
+                                    fieldLabel="Default Value">
+                                <field
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                        name="./default"
+                                        required="{Boolean}false"/>
+                            </value>
+                        </items>
+                    </basic>
+                    <patterns
+                            jcr:primaryType="nt:unstructured"
+                            jcr:title="Patterns"
+                            sling:hideResource="{Boolean}true"
+                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                    </patterns>
+                    <help
+                            jcr:primaryType="nt:unstructured"
+                            jcr:title="Help Content"
+                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured"/>
+                    </help>
+                    <accessibility jcr:primaryType="nt:unstructured"
+                                   jcr:title="Accessibility"
+                                   sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured">
```

Comment:
isn't it defined in base

Reply:
According to AFv1, the options for  assistPriorityCustom are less than what are available in base.
That is why over-ridden this item here.